xarray/backends/zarr.py

```diff
@@ -1290,6 +1457,7 @@ def open_datatree(
         storage_options=None,
         stacklevel=3,
         zarr_version=None,
+        # TODO: zarr_format
```

Comment:
Should this be added even if right now it raises a `NotImplementedError`?

Reply:
I added it and passed it through.